mautic/mautic
# [TPROD-295] Move from deprecated form authenticator to guard.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ x ]
| Automated tests included?              | [ x ] 




#### Description:
Move from deprecated form authenticator to guard.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Log in.
3. Please also check SSO login works.
